src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,94 @@
+package duke.ui;
+
+import java.util.ArrayList;
+import duke.data.task.Task;
+
+
+public class Ui {
+    public Ui() {
+    }
+    private String dottedLine = "____________________________________________________________\n";
+
+    public void showLine() {
+        System.out.println(dottedLine);
+    }
+
+    public void greet() {
+        System.out.println(dottedLine +
+                "Hello! I'm Charlie\n" +
+                "What can I do for you?\n" +
+                dottedLine);
+    }
+
+    public void exitBot() {
+        System.out.println(dottedLine +
+                "Bye. Hope to see you again soon!\n" +
+                dottedLine);
+    }
+
+    public void printMarkedMessage(Task task) {
+        System.out.println(dottedLine +
+                "Nice! I've marked this task as done:\n" +
+                task.toString() + "\n" +
+                dottedLine);
+    }
+
+    public void printUnmarkedMessage(Task task) {
+        System.out.println(dottedLine +
+                "OK, I've marked this task as not done yet:\n" +
+                task.toString() + "\n" +
+                dottedLine);
+    }
+
+    public void printAddedToListMessage(Task task, int size) {
+        System.out.println("____________________________________________________________\n" +
```

Comment:
Perhaps use the dottedLine?
```suggestion
        System.out.println(dottedLine +
```